go/registry/api/runtime.go

```diff
@@ -92,6 +92,9 @@ type ExecutorParameters struct {
 
 	// RoundTimeout is the round timeout in consensus blocks.
 	RoundTimeout int64 `json:"round_timeout"`
+
+	// MaxMessages is the maximum number of messages that can be emitted by the runtime.
```

Comment:
Maybe mention in the comment that this is the maximum number of messages _per round_.

Reply:
Good idea.